src/analytics/gtag.js

```diff
@@ -101,7 +102,7 @@ const trackExploredUsageClick = (link) => {
 }
 // tracks internal and external link clicks
 const trackLink = (link) => {
-  if (link.startsWith('http')) {
+  if (typeof link === 'string' && link.startsWith('http')) {
```

Comment:
Although this fix has already been added the pending PR #369 (to prevent an error encountered during the development), I've also added here for the same reason.